bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/ProperMotion.scala

```diff
@@ -11,16 +15,127 @@ import Scalaz._
 case class ProperMotion(
   deltaRA: RightAscensionAngularVelocity,
   deltaDec: DeclinationAngularVelocity,
-  epoch: Epoch = Epoch.J2000)
+  epoch: Epoch = Epoch.J2000) {
+
+  /** Coordinates at the current time. */
+  def calculateAt(t: Target, i: Instant): Coordinates =
+    plusYears(t, epoch.untilInstant(i))
```

Comment:
We should be able to find some examples online and at least do a handful of sanity checks.  Barnard's Star has a high proper motion for example.